casts/texts/src/UIExpandableText/UIExpandableText.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { View } from 'react-native';
+import { makeStyles, useTheme, UILabel, ColorVariants } from '@tonlabs/uikit.themes';
+import { TouchableOpacity } from '@tonlabs/uikit.controls';
+import { uiLocalized } from '@tonlabs/localization';
+import type { UIExpandableTextProps } from './types';
+import { UILayout } from '../constants';
+
+function UIExpandableTextImpl(props: UIExpandableTextProps) {
+    const { numberOfLines, testID } = props;
+    const theme = useTheme();
+    const styles = useStyles(theme);
+
+    const [isExpanded, setIsExpanded] = React.useState<boolean>(false);
+
+    const onExpand = React.useCallback(() => {
+        setIsExpanded(true);
+    }, []);
+
+    return (
+        <View style={styles.container} testID={testID}>
+            <UILabel {...props} numberOfLines={isExpanded ? undefined : numberOfLines} />
```

Comment:
You well have two `testID` props in this render. One for <View /> and one for <UILabel />.
Consider changing to:
```suggestion
        <View style={styles.container} testID={testID}>
            <UILabel {...rest} numberOfLines={isExpanded ? undefined : numberOfLines} />
```
where `rest` is taken from
```
const { numberOfLines, testID, ...rest } = props;
```

Reply:
Actually... do we need to mark View itself with the `testID`, but not a `label` itself?
If so, probably we'll also need to create an independent testID for UILabel, such as
```
testID={`UIExpandableText.Label:${testID}}
```

To let Detox find it.